docs/UserGuide.md

```diff
@@ -101,6 +137,10 @@ Prefix | Explanation
 
 * You can enter your book information in any order you like.<br>
   e.g. If our command instructions specify `n/BOOK_TITLE tp/TOTAL_PAGES`, `tp/TOTAL_PAGES n/BOOK_TITLE` is also acceptable.
+  
+* The `INDEX` will **always refer to the current shown list of books, be it in the detailed view, or the list view.**
```

Comment:
Maybe don't use terms detailed view or list view? Just something like "The `INDEX` of a book changes based on the number of books currently shown on the screen. For example, the first book will have `INDEX` 1, second book `INDEX` 2..."